web-api/src/models.rs

```diff
@@ -146,7 +146,7 @@ impl TryFrom<Value> for DocumentProperty {
 
     fn try_from(mut property: Value) -> Result<Self, Self::Error> {
         match &mut property {
-            Value::Bool(_) | Value::Number(_) => {}
+            Value::Bool(_) | Value::Number(_) | Value::Null => {}
```

Comment:
It would be better to have this as a filter in the deserialization step because internally we still do not support null values. But this can be done in a followup PR.

Reply:
Additionally this implicitly makes it now possible to set a properties value to null using the `PUT /documents/{document_id}/properties/{property}` endpoint, which is not correct and should either fail or implicitly transform it into a `DELETE /documents/{document_id}/properties/{property}`.